commerce/domain/orchestrators/classes/CartCalculateExecutorForUnitTest.cls

```diff
@@ -0,0 +1,25 @@
+/**
+* @description Sample mock executor for unit testing the various calculators within Cart Calculate API
```

Comment:
@pplat23 should we go with "for unit testing a custom orchestrator within Cart Calculate API" ?